magefiles/magefile.go

```diff
@@ -0,0 +1,243 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/magefile/mage/sh"
+	"github.com/redhat-appstudio/e2e-tests/pkg/utils"
+	"k8s.io/klog/v2"
+)
+
+var (
+	requiredBinaries = []string{"jq", "kubectl", "oc", "yq", "jq", "git"}
```

Comment:
ginkgo should not be here?

Reply:
It shouldn't, because ginkgo is being installed (or checked that it is installed) [here](https://github.com/redhat-appstudio/e2e-tests/pull/138/files#diff-d7a1577b2d546e34aaf816bdd1b83edef28a66a555f049767bafa83268b7efbbR159).
But I just noticed, that `jq` is in the list twice :) I will remove it in one of the follow-up commits/PRs